src/database/db.js

```diff
@@ -11,4 +11,43 @@ if(process.env.NODE_ENV === 'development'){
 const pgp = require('pg-promise')();
 const db = pgp( connectionString );
 
-module.exports = db;
+const getAllTodos = () => {
+  return db.any(`SELECT id, description FROM todos `)
+}
+
+
+const getCompletedTodos = () => {
+  return db.any(`SELECT id, description FROM
+    todos`)
+
+}
+
+
+const completeOneTodo = () => {
+  return db.any(`UPDATE todos
+    SET name = completed
+    WHERE id = $1
+    RETURNING *`))
+
+}
+
+const addOneTodo = () => {
+  return db.one(`INSERT INTO todos
+    (name, description)
+    VALUES
+    ($1, $2)
+    RETURNING *`)
+}
+
+
+
+
+
+
+
+
+
+
+
+
+module.exports = db, getAllTodos, getCompletedTodos,completeOneTodo,addOneTodo   ;
```

Comment:
don't need to export `db` here, only the functions, and wrap them up in some curly boys...
`{getAllTodos, getCompletedTodos, completeOneTodo, addOneTodo}`

Reply:
also the specs said to add these functions to the actions folder, either as one actions.js file or as separate files for each function

Reply:
All checks have been resolved. As far as actions folder there isn't one but i've created on in the src folder.